feat: drag & drop blocks sorting (#1184)
plentymarkets/plentyshop-pwa: docs/changelog/changelog_en.md

- Implemented `'frontend:productLoaded': { product: Product };` event which is triggered when a product gets loaded on the product page.
- Updated `frontend:removeFromCart` event to include deleted cartItem.

### New

> - Added drag-and-drop functionality for the homepage blocks & slides in context menu.

Author: ```suggestion
- Drag-and-drop functionality has been integrated for both blocks and banner context menu slides.
```
